dbConfig.js

```diff
@@ -0,0 +1,11 @@
+require("dotenv").config();
+
+const { Pool } = require("pg");
+const isProduction = process.env.NODE_ENV === "production";
+const connectionString = `postgresql://${process.env.DB_USER}:${process.env.DB_PASSWORD}@${process.env.DB_HOST}:${process.env.DB_PORT}/${process.env.DB_DATABASE}`;
```

Comment:
Can we store the DB details in one place? Right now it's in config.json and the .env file?

Reply:
I would have used the config.json however it is missing one detail that I need to pull and I don't want to mess with it until I understand it's use better.  The other db access and information storage should be benign and I plan to merge the two in refactoring later.